dashboard/views.py

```diff
@@ -15,8 +15,8 @@ def get_context_data(self, **kwargs):
             num_requests = ProjectUserMembership.objects.awaiting_authorisation(self.request.user).count()
             context['project_user_requests_count'] = num_requests
 
-        if self.request.user.has_perm('project.add_project'):
-            num_requests = Project.objects.awaiting_approval(self.request.user).count()
-            context['project_application_count'] = num_requests
+        # if self.request.user.has_perm('project.add_project'):
+        #     num_requests = Project.objects.awaiting_approval(self.request.user).count()
+        #     context['project_application_count'] = num_requests
```

Comment:
Why has this logic been disabled?

Reply:
When splitting ProjectRequests and SystemAllocationRequests into separate objects, for institutions where multiple SystemAllocationRequests can exist, the idea of an unapproved project is no longer well-defined.

In principle it would be possible to keep the same behaviour for institutions that don't separate these concepts, but is it valuable information for the user to have a notification-style badge to tell them what they already know—the allocations/projects have yet to be approved? Seems like excess visual noise.

Reply:
On the current system, when a project is approved, this triggers the process of creating a project in LDAP. Now that ProjectRequests and SystemAllocationRequests have been separated, does this effect the process of projects being created in LDAP?

Reply:
The trigger has been moved to the allocation, but should otherwise function as before. When a SystemAllocationRequest is approved, this triggers a project to be created in LDAP using update_openldap_project()